src/core/mantle/types/appearance.ts

```diff
@@ -139,6 +140,13 @@ export type ResourceAppearance = {
   clampToGround?: boolean;
 };
 
+export type RasterAppearance = {
+  zIndex?: number;
```

Comment:
```suggestion
```

Layer order can be z-index so it can be deleted

Reply:
[fix: remove zindex](https://github.com/reearth/reearth-web/pull/387/commits/a272691bbad01b41f36abe836cdb3480091c0b06)